Gradle Task quarkusRemoteDev not working with version 2.6.3

### Describe the bug

`./gradlew quarkusRemoteDev` not working on Quarkus version 2.6.3 (also same on 2.6.1 and 2.6.2)

### Expected behavior

`./gradlew quarkusRemoteDev` runs quarkus in remote dev mode

### Actual behavior

```
18:27:54: Executing 'quarkusRemoteDev'...

> Task :compileJava
> Task :processResources
> Task :classes
> Task :quarkusRemoteDev FAILED
3 actionable tasks: 3 executed

FAILURE: Build failed with an exception.

* What went wrong:
A problem was found with the configuration of task ':quarkusRemoteDev' (type 'QuarkusRemoteDev').
  - In plugin 'io.quarkus' type 'io.quarkus.gradle.tasks.QuarkusRemoteDev' property 'quarkusDevConfiguration' doesn't have a configured value.
    
    Reason: This property isn't marked as optional and no value has been configured.
    
    Possible solutions:
      1. Assign a value to 'quarkusDevConfiguration'.
      2. Mark property 'quarkusDevConfiguration' as optional.
    
    Please refer to https://docs.gradle.org/7.3.3/userguide/validation_problems.html#value_not_set for more details about this problem.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 1s
18:27:56: Execution finished 'quarkusRemoteDev'.
```


### How to Reproduce?

Steps to reproduce the behaviour:

1. Download and unpack empty project with gradle from code.quarkus.io
2. Run `./gradlew quarkusRemoteDev` inside the folder

### Output of `uname -a` or `ver`

Microsoft Windows [Version 10.0.18363.2037]

### Output of `java -version`

openjdk version "11.0.13" 2021-10-19

### GraalVM version (if different from Java)

OpenJDK 64-Bit Server VM GraalVM CE 21.3.0 (build 11.0.13+7-jvmci-21.3-b05, mixed mode, sharing)

### Quarkus version or git rev

2.6.3

### Build tool (ie. output of `mvnw --version` or `gradlew --version`)

Gradle 7.3.3

### Additional information

_No response_

--- a/devtools/gradle/gradle-application-plugin/src/main/java/io/quarkus/gradle/QuarkusPlugin.java
+++ b/devtools/gradle/gradle-application-plugin/src/main/java/io/quarkus/gradle/QuarkusPlugin.java
@@ -205,7 +205,10 @@ public class QuarkusPlugin implements Plugin<Project> {
                                 quarkusGenerateCodeTests);
                         task.setQuarkusDevConfiguration(devModeConfiguration);
                     });
-                    quarkusRemoteDev.configure(task -> task.dependsOn(classesTask, resourcesTask));
+                    quarkusRemoteDev.configure(task -> {
+                        task.dependsOn(classesTask, resourcesTask);
+                        task.setQuarkusDevConfiguration(devModeConfiguration);
+                    });
                     quarkusTest.configure(task -> {
                         task.dependsOn(classesTask, resourcesTask, testClassesTask, testResourcesTask,
                                 quarkusGenerateCode,

